src/sentry/consumers/__init__.py

```diff
@@ -75,6 +76,10 @@ def multiprocessing_options(
 
 
 # consumer name -> consumer definition
+# XXX: default_topic is needed to lookup the schema even if the actual topic name has been
```

Comment:
for now, can we rename this to `logical_topic`?

Reply:
Are you wanting to move everything in Sentry to reference the logical topic like Snuba does? I intentionally called this default rather than logical because that's not how the codebase works right now and I thought it might be misleading.

Reply:
i'm just bikeshedding the name. eventually yes, not sure but i thought it'd be clearer to point out that this is like logical-topic in snuba. default topics are defined in `conf/server.py` to me.

Reply:
I think I'd prefer to leave it right now as it may be more confusing for others - but I think we should address this fairly soon and probably move to "logical" when we do.